tests/readPreference/readpreference-set_state_error-001.phpt

```diff
@@ -53,4 +62,8 @@ OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 MongoDB\Driver\ReadPreference initialization requires "maxStalenessSeconds" integer field to be >= 90
 OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 MongoDB\Driver\ReadPreference initialization requires "maxStalenessSeconds" array field to not be present with "primary" mode
```

Comment:
`"maxStalenessSeconds" array field` looks to be old copypasta from the `tags` exception. Would you mind correcting this in a separate commit? I don't think we need to track it in an issue.